.ci_scripts/oned.sh

```diff
@@ -21,10 +21,14 @@ mkdir ~/.one
 echo 'oneadmin:opennebula' > ~/.one/one_auth
 
 # Enable dummy drivers
-sudo chmod o+w /etc/one/oned.conf
+sudo chmod o+wr /etc/one/oned.conf
 sudo chmod o+w /etc/one/monitord.conf
+sudo chmod 777 /etc/one
```

Comment:
I am not sure this is correct.

This is quite dangerous to configure this file with such rights.

Is it the same on prod ?

Mayby o+x is enough

Reply:
No, for production the permission for `oned.conf` should be:

```
-rw-r----- 1 root oneadmin 57318 Oct 21 10:17 /etc/one/oned.conf
```

The problem is that the CI tests run under an unprivileged user, so in order to properly carry the required modifications we need write and execution permissions. I added that to ensure that we avoid permissions related errors, as we're running the tests in a controlled scenario. IMHO changing the configuration file shouldn't lead to any security issues in this specific environment, but please let me know if you think otherwise and I'll try to address any possible issue.

Reply:
I understand CI constraint. I just wanted to be sure it is not the same rule in production :).

Thanks